base_ubl_payment/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# © 2016-2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Base UBL Payment",
+    "version": "14.0.1.0.0",
+    "category": "Hidden",
+    "license": "AGPL-3",
+    "summary": "Payment-related code for Universal Business Language (UBL)",
+    "author": "Akretion,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/edi",
+    "depends": ["account_payment_unece", "base_ubl"],
+    "external_dependencies": {"python": ["lxml"]},
```

Comment:
@astirpe is there any reason that you added this here: https://github.com/OCA/edi/pull/155/files#diff-304f9a575a9d52d595fac0aa80402d1e589ef778600d081546c20ff8468a0fc1R13? `lxml` is a standard Odoo requirement, so I do not expect it to be listed as an addon dependency. I would suggest to remove it if it is redundant indeed.